tests/unit/components/Items/Question/Body.spec.js

```diff
@@ -288,6 +288,23 @@ describe("Body.vue", () => {
           .classes()
       ).toContain("bg-red-500");
     });
+
+    it("highlights options gray for survey question answers", async () => {
+      const submittedAnswer = 0;
+      const correctAnswer = 1;
+      await wrapper.setProps({
+        submittedAnswer: submittedAnswer,
+        correctAnswer: correctAnswer,
+        isAnswerSubmitted: true,
+        isSelectedItemSurveyQuestion: true,
```

Comment:
realised that `isSelectedItemSurveyQuestion` is not the right prop name for `Body.vue` as there is no idea of selected item in `Body.vue` - best to rename it to something like `isSurveyQuestion`